### ISSUE TITLE
Polyline popups

### ISSUE THREAD

**Issue by Sarrasor** [indoorequal/vue-maplibre-gl#73]
Hello. Thanks for the library!

I have a question regarding popups for polylines. I have multiple polylines along with some markers and need to have popups on each of them. For example:

![example_popup](https://github.com/user-attachments/assets/5e1ef5d6-84f4-4e7a-a2f5-1c7ef160a092)

Both polylines and markers will be referred to as jobs below. Polyline jobs have the type `STREET` and marker jobs have the type `LOCATION`

I haven't found any documentation regarding that for your library, so I have invented a temporary solution to achieve the desired result. I am creating a standalone popup, and accessing Maplibre map object directly. It works, but is very ugly:

```
<mgl-popup v-if="selectedJob" :coordinates="popupCoordinates">
    <h1 class="text-lg font-bold">Example popup</h1>
 </mgl-popup>

...

onMounted(() => {
    if (mapRef.value) {
        const map = mapRef.value.map;

        map.on('load', () => {
            map.on('click', (e) => {
                const features = map.queryRenderedFeatures(e.point, {
                    layers: props.runInput.jobs.filter(job => job.jobType !== 'LOCATION').map(job => job.id)
                });

                if (features.length > 0) {
                    const clickedJob = props.runInput.jobs.find(job => job.id === features[0].layer.id);
                    if (clickedJob) {
                        selectedJob.value = clickedJob;
                        popupCoordinates.value = e.lngLat.toArray();
                    }
                } else {
                    selectedJob.value = null;
                }
            });
        });
    }
});
``` 

Another issue with that method is warnings are printed in the console on each popup click. It seems a polyline popup wants to have a marker associated with it:

```
[Vue warn]: injection "Symbol(marker)" not found. 
  at <MglPopup key=0 coordinates= 
...
```

Do you have any better way to do that?

Another related question is how to make popups displayed on hover instead of on click? There is a Maplibre [example](https://maplibre.org/maplibre-gl-js/docs/examples/popup-on-hover/) on how to do that, but I can't seem to figure out how to do that with your library

**Comment by francois2metz** [comment]
Hi,

Your issue is more a question. Without the code and time it's difficult to answer it. Would be happy to help privately.

Anyway, I fixed the runtime warning, thanks for finding it. The fix has been released in the v7.6.1 version.

**Comment by Sarrasor** [comment]
Thanks for prompt answer and fixes!

Does that mean the library does not support popups for polylines and popus on hovers? If so, I would be happy to implement it
